package.json

```diff
@@ -11,8 +11,10 @@
     "lint": "eslint src"
   },
   "dependencies": {
+    "html5-websocket": "^2.0.2",
     "isomorphic-fetch": "^2.2.1",
     "lodash.zipobject": "^4.1.3",
+    "reconnecting-websocket": "^3.2.2",
     "ws": "^3.2.0"
```

Comment:
Do we still need this dependency?

Reply:
`reconnecting-websocket` is needed but I was able to remove `ws` as it is already a dependency of `html5-websocket` 😃

Reply:
Yep, perfect